qemu/tests/balloon_check.py

```diff
@@ -355,6 +356,94 @@ def get_win_mon_free_mem(self, session):
         else:
             self.test.fail("Failed to get windows guest free memory")
 
+    def run_balloon_service(self, session, win_drive_letter):
+        """
+        Start balloon service in windows guest
+
+        :param session: shell Object
+        :param win_drive_letter: virtio-win disk volume letter.
+        :return: cmd execution status and output
+        """
+        try:
+            start_cmd = self.params["run_balloon_service"] % win_drive_letter
+            status, output = session.cmd_status_output(start_cmd)
+            return (status, output)
+        except ShellCmdError:
```

Comment:
ShellCmdError is not necessary here, as session.cmd_status_output(start_cmd) can handle error info.
If use except ShellCmdError, session.cmd(start_cmd) is okay.

Reply:
You're right, will modify it.
Thanks